stream-chat-android-compose-sample/src/main/java/io/getstream/chat/android/compose/sample/ui/channel/ChannelInfoActivity.kt

```diff
@@ -68,8 +68,7 @@ class ChannelInfoActivity : BaseConnectedActivity() {
          * @param channelId The ID of the channel for which the pinned messages are shown.
          */
         fun createIntent(context: Context, channelId: String) =
-            Intent(context, ChannelInfoActivity::class.java)
-                .putExtra(KEY_CHANNEL_ID, channelId)
+            Intent(context, ChannelInfoActivity::class.java).putExtra(KEY_CHANNEL_ID, channelId)
```

Comment:
Can we revert this change?
Having `extra` in a new line helps us with PR Conflicts in case new extras need to be added